### PR TITLE
Add chapter 1

### PR DESCRIPTION
**Pull request by canyon289**
Also add gitignore and docker file because I couldn't get stan and theano to install on my mac for some reason

### CODE REVIEW

**Comment by aloctavodia** [comment]
Thanks @canyon289. This is using the old version of bambi. Could you try installing bambi with pip? (as the conda package seems to be broken).

**Comment by canyon289** [comment]
Yes will do so in next 24 hours

**Comment by canyon289** [comment]
Updated 
image

File: ROS/Chapter_1.ipynb [hunk 1]
```diff
@@ -0,0 +1,657 @@
+{
```
Comment (aloctavodia) [inline ROS/Chapter_1.ipynb:1]: <p>I still feel uneasy with this warning, haha</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/Bambi_resources/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (aloctavodia) [inline ROS/Chapter_1.ipynb:1]: <p>what about making the data available from the bambi repo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/Bambi_resources/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (canyon289) [reply ROS/Chapter_1.ipynb:1]: <p>Ill figure out how to get rid of it</p>

Comment (canyon289) [reply ROS/Chapter_1.ipynb:1]: <p>I was debating this but not sure what to do. Here's factors I have</p><ul><li>Aki is still changing around the data in preparation for book</li><li>Why duplicate if its already being maintained somewhere else?</li></ul><p> </p><p>That being said not opposed maybe we should copy it here? </p>

Comment (aloctavodia) [reply ROS/Chapter_1.ipynb:1]: not a big deal is more my problem really, haha.

Comment (aloctavodia) [reply ROS/Chapter_1.ipynb:1]: Yeah I think having a copy is a better option. We may want to do it, once the book is published. So we can merge this.

Comment (canyon289) [reply ROS/Chapter_1.ipynb:1]: Do we want to merge and then update when book is out? I'm fine with whatever, just want to know so I can create another PR with just the container whatnot so I'm not branching off of branches as I go through the chapters

**Comment by aloctavodia** [comment]
Thanks Ravin!

**Pull request merged**